client/src/components/lobby.js

```diff
@@ -3,10 +3,12 @@ var OpenStory = require('./OpenStory')
 var Accordion = require('./Accordion')
 var GameBoard = require ('./games.js')
 var API = require('../lib/api')
+
 module.exports = class Lobby extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
+      currentUser: this.props.route.user,
```

Comment:
let's leave this out of the state, and it should be an object being passed from props